docs/metrics/kubernetes-metrics.md

```diff
@@ -247,120 +248,3 @@ To view metrics, open your browser to:
 
 Where `SUMOLOGIC_HELM_CHART_NAMESPACE` is the namespace where the Sumo Logic Kubernetes Collection Helm Chart is installed and
 `SUMOLOGIC_HELM_CHART_RELEASE_NAME` is the the release name of the Sumo Logic Kubernetes Collection Helm Chart in your cluster.
-
-## Metrics created by Prometheus rules (default-metrics)
-
-These metrics are produced by Prometheus rules at ingest time — they are not available on an exporter, Prometheus creates them during scrape activities. Rules-created metrics are sent with the remote write API.
-
-You can collect metrics created by Prometheus Operator rules in managed Kubernetes environments, such as GKE, EKS, or AKS, or in a Kubernetes deployment you manage yourself.
-
-These metrics can be found under the **default-metrics** source in Sumo Logic.
-
-### Metrics to collect
-
-* :kube_pod_info_node_count:
```

Comment:
Hi @dmolenda-sumo , to confirm, we don't need to display these metrics in the docs any longer? Are they obsolete?